format.md

```diff
@@ -49,51 +49,53 @@ The terminology for array types in this section is based on the
 GeoArrow proposes a packed columnar data format for the fundamental geometry
 types, using packed coordinate and offset arrays to define geometry objects.
 
-The inner level is always a FixedSizeList array storing the interleaved
-coordinate values (`[x, y, x, y, ...]`, physically stored as a single array
-of length `n_dim * n_coords`). For any geometry type except Point, this inner
-level is nested in one or multiple
+The inner level is always a Struct array storing the
+coordinate values (`x: [x, x, x, x], y: [y, y, y, y]`, physically stored as
+one array per dimension with the name of each child array corresponding to
+the dimension it represents (i.e., x, y, z, or m). For any geometry type except
+Point, this inner level is nested in one or multiple
 [variable sized list](https://arrow.apache.org/docs/format/Columnar.html#variable-size-list-layout)
 arrays. In practice, this means we have additional arrays storing the offsets
 that denote where a new geometry, a new geometry part, or a new polygon ring
 starts.
 
-**Point**: `FixedSizeList<double>[n_dim]`
+**Point**: `Struct<x: double, y: double, [z: double, [m: double>]]`
```

Comment:
In this case, is it necessary for m to be a `double` at all? For example, if I have a Line representing the path of a ship on the ocean, I might want want me to be a `Date64`.

Reply:
Or more ambitiously but still grounded in the actual example I have in mind, I might want m to be `struct<time: Date64, windspeed: float64, winddirection: dictionary, notes: string>`. Or I guess there's `t` as well. I might also just be asking for arbitrary keys in the array where only `x`, `y`, and `z` (?or not z?) are strictly typed.